src/modules/form/fieldBuilder.php

```diff
@@ -568,717 +568,712 @@ public function renderLabel($options = array()){
+  }
+
+  /**
+   * [Render Helper] Render a plaintext field
+   * @return string
+   */
+  private function __render_plaintext(){
+    $value      = $this->getFieldOption('value');
+    $formFields = $this->formFields;
+    $value      = preg_replace_callback('/\{(.*?)\}/', function ($m) use ($formFields){
+      $field = $formFields->getField($m[1]);
+      $value = !is_empty($field->renderedValue) ? $field->renderedValue : $field->value;
+      return $value;
+    }, $value);
+
+    return $value;
+  }
+
+  /**
+   * [Render Helper] Render a password field
+   * @return string
+   */
+  private function __render_password(){
+    // Render the password field
+    if(!isset($this->renderOptions['placeholder'])) $this->renderOptions['placeholder'] = 'Password';
+    $output = $this->__render_input();
+
+    // Render the password confirmation field
+    $this->renderOptions['name']    = $this->getFieldOption('name').'_confirm';
+    $this->renderOptions['fieldID'] = $this->getFieldOption('fieldID').'_confirm';
+    $this->renderOptions['placeholder'] = $this->getFieldOption('placeholder').' Confirmation';
+    $output .= $this->__render_input();
+
+    return $output;
+  }
+
+  /**
+   * [Render Helper] Render a textarea field
+   * @return string
+   */
+  private function __render_textarea(){
+    return sprintf('<textarea %s%s>%s</textarea>',
+      $this->buildFieldAttributes(),
+      (!is_empty($this->getFieldOption('placeholder')) ? ' placeholder="'.$this->getFieldOption('placeholder').'"' : ''),
+      $this->getFieldOption('value')
+    );
+  }
+
+  /**
+   * [Render Helper] Render a boolean field
+   * @return string
+   */
+  private function __render_boolean(){
+    $options = $this->getFieldOption('options');
+    $type    = isset($options['type']) ? $options['type'] : 'select';
+
+    // Blank the options
+    $this->renderOptions['options'] = array();
+
+    // Determine labels
+    $no  = isset($options['labels']) ? array_shift($options['labels']) : 'No';
+    $yes = isset($options['labels']) ? array_shift($options['labels']) : 'Yes';
+
+    switch(trim(strtolower($type))){
+      case 'check':
+      case 'checkbox':
+        $this->renderOptions['options'][1] = 'Yes';
+        return $this->__render_checkbox();
+
+      case 'radio':
+        $this->renderOptions['options'][0] = $no;
+        $this->renderOptions['options'][1] = $yes;
+        return $this->__render_radio();
+
+      case 'select':
+      default:
+        if(isset($options['blankOption'])) $this->renderOptions['blankOption'] = $options['blankOption'];
+        $this->renderOptions['options'][0] = $no;
+        $this->renderOptions['options'][1] = $yes;
+        return $this->__render_select();
+    }
+  }
+
+  /**
+   * [Render Helper] Render a password field
+   * @return string
+   */
+  private function __render_delete(){
+    $this->field['type'] = 'submit';
+    $this->field['name'] = "delete";
+    return  $this->__render_input();
```

Comment:
Was this tested? I don't think this will fix the issue. My current local branch looks like this:
```                $type = $this->field['type'];
		$name = $this->field['name'];
                $this->field['type'] = 'submit';
		$this->field['name'] = "delete";
                $output = $this->__render_input();
                $this->field['type'] = $type;
                $this->field['name'] = $name;
```
I've also tried it without preserving either the type or name fields. I'm not seeing  change locally. 

If you tested this, and it was working, i would like to know your procedure so I can reproduce it.